test/utils.ts

```diff
@@ -99,11 +101,13 @@ export async function getFakeEmailVerifier(result: boolean) {
       type: 'function',
     },
   ])
-  fake.verifyProof.returns(result)
   return fake
 }
 
-export function getFakeEmailProof(nullifier: number, domain: string) {
+export function getFakeEmailProof(
+  nullifier: number,
+  domain: string
+): EmailProofStruct {
```

Comment:
You shouldn't need to specify the return type if it's compatible with the struct